### PR TITLE
Fix channel mismatch during normalization

### PR DESCRIPTION
**Pull request by damster101**
Previously, all channels in file b were normalized to the second channel in file a, instead of the correct, corresponding channel. This commit fixes that.

### CODE REVIEW

File: cpp/zimt/compare.cc [hunk 1]
```diff
@@ -246,7 +246,7 @@ int Main(int argc, char* argv[]) {
       for (size_t channel_index = 0; channel_index < file_b->Info().channels;
            ++channel_index) {
         const EnergyAndMaxAbsAmplitude new_energy_and_max_abs_amplitude =
-            NormalizeAmplitude(file_a_max_abs_amplitude,
```
Comment (zond) [inline cpp/zimt/compare.cc:249]: Nice, but now you can remove `file_a_max_abs_amplitude` completely, right?

Comment (damster101) [reply cpp/zimt/compare.cc:249]: Yes, I do think so. Good catch.

Comment (zond) [reply cpp/zimt/compare.cc:249]: Great!

If you handle https://opensource.google/documentation/reference/cla, I can patch this PR to remove the variable completely if you don't want to do it yourself.

Comment (damster101) [reply cpp/zimt/compare.cc:249]: I tried to patch the PR myself, and I think it worked. Learning a lot here :)

**Comment by zond** [comment]
Thanks! It'll merge once the tests have passed.

**Comment by damster101** [comment]
Why doesn't the build check complete...

**Comment by damster101** [comment]
oops, now someone has to approve again... thought merging with main might trigger the build check to go through

**Comment by zond** [comment]
It's running now - let's hope it works :D

**Pull request merged**